common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -199,6 +199,13 @@
     return get_value('SHOW_LANGUAGE_SELECTOR', settings.FEATURES.get('SHOW_LANGUAGE_SELECTOR', False))
 %></%def>
 
+<%def name="show_footer_language_selector(api_setting)"><%
```

Comment:
@andy-armstrong I'm not sure that this is any more edx-specific than the `show_language_selector` def. The footer language selector can be used by any openedx instance. All footer templates support it.

The `api_setting` parameter allows us to override what's in settings when requesting the footer via the API, so that we may explicitly decide to include/exclude it on certain pages regardless of what we're doing for the broader LMS.